packages/llmd-serving/src/deployments/deploy.ts

```diff
@@ -1,15 +1,34 @@
-import { k8sCreateResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { k8sCreateResource, k8sUpdateResource } from '@openshift/dynamic-plugin-sdk-utils';
 import { applyK8sAPIOptions } from '@odh-dashboard/internal/api/apiMergeUtils';
 import {
   type WizardFormData,
   ModelLocationType,
   ModelLocationData,
 } from '@odh-dashboard/model-serving/types/form-data';
+import * as _ from 'lodash-es';
+import { k8sMergePatchResource } from '@odh-dashboard/internal/api/k8sUtils';
+import { HardwareProfileConfig } from '@odh-dashboard/internal/concepts/hardwareProfiles/useHardwareProfileConfig';
 import { applyHardwareProfileConfig, applyReplicas } from './hardware';
-import { applyModelEnvVars, applyModelArgs, applyModelLocation } from './model';
 import { setUpTokenAuth } from './deployUtils';
+import {
+  applyModelEnvVars,
+  applyModelArgs,
+  applyModelLocation,
+  applyAiAvailableAssetAnnotations,
+  applyDisplayNameDesc,
+  applyDashboardResourceLabel,
+} from './model';
+import { AvailableAiAssetsFieldsData } from '../../../model-serving/src/components/deploymentWizard/fields/AvailableAiAssetsFields';
 import { LLMD_SERVING_ID } from '../../extensions/extensions';
-import { LLMdDeployment, LLMInferenceServiceKind, LLMInferenceServiceModel } from '../types';
+import {
+  LLMdContainer,
+  LLMdDeployment,
+  LLMInferenceServiceKind,
+  LLMInferenceServiceModel,
+} from '../types';
+import { RuntimeArgsFieldData } from '../../../model-serving/src/components/deploymentWizard/fields/RuntimeArgsField';
+import { EnvironmentVariablesFieldData } from '../../../model-serving/src/components/deploymentWizard/fields/EnvironmentVariablesField';
+import { CreateConnectionData } from '../../../model-serving/src/components/deploymentWizard/fields/CreateConnectionInputFields';
```

Comment:
Using the package imports would be good to do

```suggestion
import { RuntimeArgsFieldData } from '@odh-dashboard/model-serving/src/components/deploymentWizard/fields/RuntimeArgsField';
import { EnvironmentVariablesFieldData } from '@odh-dashboard/model-serving/src/components/deploymentWizard/fields/EnvironmentVariablesField';
import { CreateConnectionData } from '@odh-dashboard/model-serving/src/components/deploymentWizard/fields/CreateConnectionInputFields';
```